docs/Gemfile

```diff
@@ -0,0 +1,7 @@
+source 'https://rubygems.org'
```

Comment:
You need to watch this file with dependabot.

Reply:
Done, but please review...